### ISSUE TITLE
Touch control with stylus

### ISSUE THREAD

**Issue by DvaMishkiLapa** [alexbatalov/fallout2-ce#286]
I have a screen that supports one touch, as well as a stylus. By default, F2 seems to run with mobile device control (moving the cursor with your finger).
How do I switch to the control mode the same as a mouse? Where I pressed with the stylus, that's where the mouse should be.

I have been using the `armv7` source build on Alpine Linux.

**Comment by roginvs** [comment]
As workaround I did this https://github.com/roginvs/fallout2-ce/commit/f6a685ffc02aec8ffd2315bde127ef22382f2f45 but I am sure that it can be done better

**Comment by widardd** [comment]
> As workaround I did this [roginvs@f6a685f](https://github.com/roginvs/fallout2-ce/commit/f6a685ffc02aec8ffd2315bde127ef22382f2f45) + [roginvs@c546d14](https://github.com/roginvs/fallout2-ce/commit/c546d1476e1098235e6706e575e77bb6cb8e3020) but I am sure that it can be done better

Your built APK just crashes for me.

**Comment by roginvs** [comment]
@widardd Which one? Did you try to run .apk from organization? https://github.com/fallout2-ce/fallout2-ce/releases

**Comment by widardd** [comment]
@roginvs Yup! But honestly, I discovered a talented Ukranian and his project FO2.EXE. I'm fine now.😅
